playbooks/cloud-provider-openstack-acceptance-test-csi-manila/run.yaml

```diff
@@ -140,12 +140,16 @@
           metadata:
             name: csi-manila-secrets
             namespace: default
+          # Use the devstack ``demo`` user in the ``demo`` project rather than ``admin``
+          # since regular users should be able to use the cloud-provider-openstack
+          # APIs without OpenStack administrative privileges.  Devstack sets up
+          # up the same password for both users.
```

Comment:
It's good to make the user configurable in `install-devstack` role. While the role doesn't support it currently, I'm OK with this change.